api/capi/src/nntrainer.cpp

```diff
@@ -968,6 +968,82 @@ int ml_train_dataset_destroy(ml_train_dataset_h dataset) {
   return status;
 }
 
+int ml_train_model_get_input_dimensions(ml_train_model_h model,
+                                        ml_tensors_info_h *info) {
+  int status = ML_ERROR_NONE;
+  ml_train_model *nnmodel;
+  std::shared_ptr<ml::train::Model> m;
+  returnable f;
+
+  check_feature_state();
+
+  ML_TRAIN_GET_VALID_MODEL_LOCKED(nnmodel, model);
+  ML_TRAIN_ADOPT_LOCK(nnmodel, model_lock);
+  m = nnmodel->model;
+  if (m == NULL) {
+    return ML_ERROR_INVALID_PARAMETER;
+  }
+
+  std::vector<ml::train::TensorDim> dims = m->getInputDimension();
+
+  status = ml_tensors_info_set_count(*info, dims.size());
```

Comment:
The interface expects the allocation of info object by this function. 
The allocation of the ml_tensors_info_h must be done in this function than by the caller (like in the unittest).

Reply:
Done